.travis.yml

```diff
@@ -8,8 +8,19 @@ cache:
   directories:
     - $HOME/.composer/cache
 
+matrix:
+  fast_finish: true
+  include:
+    - php: 5.4
+      env: deps="low"
+    - php: 5.6
+      env: SYMFONY_VERSION="2.8.x-dev"
+    - php: 5.6
+      env: SYMFONY_VERSION="3.0.x-dev"
+
 before_install:
-  - echo "extension=amqp.so" >> `php --ini | grep "Loaded Configuration" | sed -e "s|.*:\s*||"`
+  - composer self-update
+  - if [ "$SYMFONY_VERSION" != "" ]; then composer require --dev --no-update symfony/framework-bundle:"$SYMFONY_VERSION"; fi
 
 install:
-  - composer install -n
+  - if [ "$deps" = "low" ]; then composer update --no-interaction --prefer-lowest --prefer-stable; else composer install --no-interaction; fi
```

Comment:
no need for ``--no-interaction``. Travis already sets the necessary env variable to prevent interaction